spaceship/lib/spaceship/tunes/app_submission.rb

```diff
@@ -142,7 +142,8 @@ def complete!
           )
         end
 
-        client.send_app_submission(application.apple_id, application.edit_version.version_id, raw_data_clone)
+        client.send_app_submission(application.apple_id, application.edit_version.version_id, raw_data_clone, reject_if_waiting_for_review)
```

Comment:
There isn't a reason to keep passing `reject_if_waiting_for_review` into here (as this is more business logic and less API client logic). I would take your logic you put in `send_app_submission` and actually put it into its own method in this class called `reject_if_waiting_for_review!`